lib/services/access_list.go

```diff
@@ -21,32 +21,32 @@ import (
 
 	"github.com/gravitational/trace"
 
-	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/types/accesslist"
 	"github.com/gravitational/teleport/lib/utils"
 )
 
 // AccessListsGetter defines an interface for reading access lists.
 type AccessListsGetter interface {
 	// GetAccessLists returns a list of all access lists.
-	GetAccessLists(context.Context) ([]*types.AccessList, error)
+	GetAccessLists(context.Context) ([]*accesslist.AccessList, error)
```

Comment:
Not a huge fan of the name stuttering on the calling side here - but unfortunately I don't have a good suggestion for the `AccessList` type itself.